### DISCUSSION TITLE
MVP for B2B Grocery

### DISCUSSION THREAD

**Discussion by BLR-0118** [ONDC-Official/protocol-network-extension#42] [General]
MVP (draft) for B2B Grocery is available [here](https://docs.google.com/document/d/1eV7teSyNwpSVvk6IBeUI5QStry7dWZLTv5Q58Tbd5xU/edit) for your comments

**Comment by BLR-0118** [comment]
@dashbitla-ib - for your comments

**Reply by dashbitla-ib** [reply]
Looks good Supriyo! Few suggestions added.
